airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3DestinationConfig.java

```diff
@@ -35,29 +35,28 @@
   private final S3FormatConfig formatConfig;
 
   /**
-   * The part size should not matter in any use case that depends on this constructor. So the default
-   * 10 MB is used.
+   * The part size should not matter in any use case that depends on this constructor. So the default 10 MB is used.
    */
   public S3DestinationConfig(
-                             final String endpoint,
-                             final String bucketName,
-                             final String bucketPath,
-                             final String bucketRegion,
-                             final String accessKeyId,
-                             final String secretAccessKey,
-                             final S3FormatConfig formatConfig) {
+      final String endpoint,
+      final String bucketName,
+      final String bucketPath,
+      final String bucketRegion,
+      final String accessKeyId,
+      final String secretAccessKey,
+      final S3FormatConfig formatConfig) {
```

Comment:
Make sure to run `format` before merging this PR. I think these format changes are not consistent with the current spotless setting.

Reply:
are there additional steps beyond https://docs.airbyte.io/contributing-to-airbyte/code-style ? Intellij keeps generating these changes :/

Reply:
I don't think so. Usually I just rely on `format` to do the job.

Reply:
hmmm. is there a way to format a specific module/directory? `./gradlew :airbyte-integrations:connectors:destination-s3:format` doesn't exist, and I don't think running `format` as a commit hook would be very pleasant

Reply:
Module-specific formatting task seems to have been removed. I cannot find it either. For now, maybe just run it once before merging the PR?